contrib/greetd/greetd.post-install

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# install greeter home dir
+install -d -m 755 /var/lib/greeter || :
+chown _greeter:_greeter /var/lib/greeter > /dev/null 2>&1 || :
```

Comment:
everything here is already done automatically by having system_users with this set, no?

Reply:
Afaict, no. I initially didn't have this, but the home dir was not created, so I did what gdm does

Reply:
`useradd` does not create a homedir for system users (`-r`)